apps/studio/prisma/schema.prisma

```diff
@@ -33,6 +33,16 @@ model VerificationToken {
   expires    DateTime
 }
 
+model Version {
+  id          BigInt     @id @default(autoincrement())
+  versionNum  BigInt
+  resourceId  BigInt
+  resources   Resource[]
+  blobId      BigInt
+  blob        Blob       @relation(fields: [blobId], references: [id])
+  publishedAt DateTime   @default(now())
+}
```

Comment:
suggestion(non-blocking): I think we should also add a publishedBy field to note which user performed the publish

Reply:
Qn here from Slack discussion is also separation of concerns bet versions and audit log. But I think for MVP we can keep it in Versions. Will add.